accord-core/src/test/java/accord/utils/AccordGens.java

```diff
@@ -227,7 +236,14 @@ public static Gen<Key> keysInsideRanges(Ranges ranges)
 
     public static Gen<KeyDeps> keyDeps(Gen<? extends Key> keyGen)
```

Comment:
this is used in maelstrom and `org.apache.cassandra.utils.AccordGenerators#keyDepsGen()`, need to double check the C* to make sure this change is fine.